Frontend/src/cse-ui-kit/footer/footer.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { StyledFooter, FooterProps} from './footer-Styled';
+
+type Props = {
+    children?: React.ReactElement | any;
+    className?: string;
+  } & FooterProps;
+
+export default function Footer({ children, className}: Props) {
+    return (
+        <StyledFooter className={className}>
```

Comment:
className? I dont think we use this convention for styling our css since we are using styled components